### PR TITLE
[MM-29332] Add e2e tests to renameChannelCmdF

### PR DESCRIPTION
**Pull request by simross**
**Summary**

Adds e2e tests for renameChannelCmdF.

**Ticket Link**

Fixes mattermost/mattermost-server#15810

JIRA: https://mattermost.atlassian.net/browse/MM-29332

### CODE REVIEW

**Review by reflog — APPROVED** [review]
LGTM :100:

**Review by mgdelacroix — COMMENTED** [review]
Thanks for your contribution @simross!! Great work! The code is perfect, but the branch seems to include as well tests for other commands. Could you please leave only the ones related to the `TestChannelRenameCmd` function?

**Review by lindy65 — APPROVED** [review]
Thanks @simross 👍 

Apart from your comment regarding other tests in the code @mgdelacroix, looks good to me :)

**Comment by simross** [comment]
Reverted unrelated commits.

**Comment by hanzei** [comment]
@simross Could you please resolve the merge conflict?

File: commands/channel_e2e_test.go [hunk 1]
```diff
@@ -206,3 +206,114 @@ func (s *MmctlE2ETestSuite) TestUnarchiveChannelsCmdF() {
...
+
+		s.Require().Equal(newChannelName, printedChannel.Name)
+		s.Require().Equal(newChannelDisplayName, printedChannel.DisplayName)
+
+		rchannel, err := s.th.App.GetChannel(channel.Id)
+		s.Require().Nil(err)
+		s.Require().Equal(newChannelName, rchannel.Name)
+		s.Require().Equal(newChannelDisplayName, rchannel.DisplayName)
+	})
+}
```
Comment (hanzei) [inline commands/channel_e2e_test.go:319]: Please add an empty line here to fix the linter issue

Comment (simross) [reply commands/channel_e2e_test.go]: done

**Review by mgdelacroix — APPROVED** [review]
LGTM @simross!! Thanks for the contribution! :sparkles:

**Pull request merged**